src/index.html

```diff
@@ -274,6 +291,9 @@ <h2 class="font-vag-rounded text-3xl text-geblue dark:text-white">
         <h2 class="font-vag-rounded text-3xl text-geblue dark:text-white">
           Contact
         </h2>
+        <a href="https://tradehq.com/groundedelectricalservices/enquire">
+          <button>Request a Quote</button>
+        </a>
```

Comment:
I'll check if this looks OK tonight. It might work just like this, but I'm unsure - it'll probably need some css styling